runtime/tests/runtime_integration_tests/test_certs.rs

```diff
@@ -161,7 +161,7 @@ fn get_ldev_cert(model: &mut DefaultHwModel) -> GetLdevCertResp {
     ldev_resp
 }
 
-fn get_fmc_alias_cert(model: &mut DefaultHwModel) -> GetFmcAliasCertResp {
+pub fn get_fmc_alias_cert(model: &mut DefaultHwModel) -> GetFmcAliasCertResp {
```

Comment:
Maybe just move this function to common.rs?

Reply:
#1104